interface/globals.php

```diff
@@ -316,6 +297,26 @@ function GetCallingScriptName()
     die();
 }
 
+// This will open the openemr mysql connection.
+require_once(__DIR__ . "/../library/sql.inc");
```

Comment:
My original thought is that this should break since need credentials in sqlconf.php. However it does not break since sql.inc brings in sqlconf.php. Thus to clarify things, would put the the sqlconf.php block above this line.

Reply:
If line 301 calls a file that requires library/sql.conf couldn't we remove line 309?

Reply:
yep, could do that too :)